packages/components/inputs/select-utils/src/clear-indicator/clear-indicator.tsx

```diff
@@ -1,11 +1,19 @@
-import PropTypes from 'prop-types';
+import { CSSProperties, LegacyRef } from 'react';
```

Comment:
When importing types, prefer to use `import type`

```suggestion
import type { CSSProperties, LegacyRef } from 'react';
```